fuzz/api/main.go

```diff
@@ -14,9 +15,15 @@ func Fuzz(data []byte) int {
 		return -1
 	}
 
-	// should not panic
-	baker := agent.InitializeTemplateGenerator()
-	baker.GetNodeBootstrappingCmd(&config)
+	baker, err := agent.NewAgentBaker()
+	if err != nil {
+		return -2
```

Comment:
I don't think this actually does anything? it deprioritizes invalid input but only -1/0/1 make a difference

Reply:
oops